client/boot/index.js

```diff
@@ -343,7 +343,7 @@ function reduxStoreReady( reduxStore ) {
 	}
 
 	if ( config.isEnabled( 'desktop' ) ) {
-		require( 'lib/desktop' ).init();
+		// require( 'lib/desktop' ).init();
```

Comment:
I don't get a trace in Chrome, but [here's the message](https://cloudup.com/cPn_fus279q). This is from `master` on both wp-desktop/wp-calypso.